File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
import java.util.*;
class Solution {
    public String solution(String s) {
        String[] splitS = s.split(" ");
        int max=Integer.MIN_VALUE;
        int min=Integer.MAX_VALUE;
        int a = 0;
        for(String i:splitS){
            a = Integer.parseInt(i);
            if(max<a) {
                max = a;
            }
            if(min>a)
                min = a;
        }
        return min+" "+max;
    }
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.js
function solution(s) {
    let sArr = s.split(' ').map(x=> parseInt(x))
    let max = sArr.reduce((a,b)=> Math.max(a,b))
    let min = sArr.reduce((a,b)=> Math.min(a,b))
    return `${min} ${max}`
}